app/assets/stylesheets/_conditionals.scss

```diff
@@ -0,0 +1,40 @@
+// Media query helpers. These make producing IE layouts
+// super easy.
+
+// These are desktop and down media queries
+
+// There is also a local version of this in Smartanswers.
+
+$is-ie: false !default;
+
+@mixin media-down($size: false, $max-width: false, $min-width: false) {
```

Comment:
This mixin appears to be in the components gem already, although it doesn't include the `ie-lte` and `ie` mixins further down. Can this mixin be removed?

https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/mixins/_media-down.scss#L10

Reply:
I haven't looked inside the SCSS files – it was more of a lift and shift operation without any improvements. This project is in discussions to be deprecated and I don't think it's worth investing time to improve it unless there's a mission dedicated to this. If I start looking inside the SCSS files I will most probably end up refactoring it all.

Reply:
That's fair enough then.